.github/workflows/csi_operator_ci.yaml

```diff
@@ -0,0 +1,84 @@
+name: Community Operators test
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+jobs:
+  operator_image_build:
+   name: "set env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+   outputs:
+     operator_image_for_test: "ibmcom/ibm-block-csi-operator:latest"
+
+  prepare_env:
+   name: "prepare env"
+   needs: operator_image_build
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Install dependencies
+        run: python -m pip install --upgrade pip yq
+      - name: install apt dependencies
+        run: |
+          build/ci/install_ci_dependencies.sh
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: Prepare_env
+        id: environment_setup
+        run: |
+          build/ci/prepare_env.sh
+   outputs:
+     community_operators_path: "${{ steps.environment_setup.outputs.community_operators_path }}"
+     upstream_community_operators_path: "${{ steps.environment_setup.outputs.upstream_community_operators_path }}"
+     repository_path: "${{ steps.environment_setup.outputs.repository_path }}"
+     operator_image_in_branch: "${{ steps.environment_setup.outputs.operator_image_in_branch }}"
+     operator_image_for_test: ${{ needs.operator_image_build.outputs.operator_image_for_test }}
+     csv_file:  ${{ steps.environment_setup.outputs.csv_file }}
+
+  CI-csi-demo-pr:
+    name: "community-operators fork checks"
+    needs: prepare_env
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create demo PR
+        run: |
+          build/ci/create_demo_pr.sh
+        env:
+          github_csiblock_community_operators_repository: "csiblock/community-operators"
+          community_operators_kubernetes_branch: "ibm-block-csi-update-kubernetes-${{ github.run_number }}"
+          community_operators_openshift_branch: "ibm-block-csi-update-openshift-${{ github.run_number }}"
+          github_token: ${{ secrets.CSIBLOCK_GITHUB_TOKEN }}
+          repository_path: ${{ needs.prepare_env.outputs.repository_path }}
+          operator_image_in_branch: ${{ needs.prepare_env.outputs.operator_image_in_branch }}
+          operator_image_for_test: ${{ needs.prepare_env.outputs.operator_image_for_test }}
+          csv_file: ${{ needs.prepare_env.outputs.csv_file }}
+      - name: Check if all the demo PR checks were passed
```

Comment:
consider shortening to `Assert all demo PR checks passed`

Reply:
done